packages/api/typings/app.ts

```diff
@@ -33,6 +33,8 @@ export interface QueueJob {
   retry(state?: JobRetryStatus): Promise<void>;
 
   toJSON(): QueueJobJson;
+
+  getState(): Promise<Status | 'stuck' | 'waiting-children' | 'unknown'>;
```

Comment:
Apparently this is the full intersection of states 🤷‍♂️  - idk if this is going to cause problems, any advice here would be greatly appreciated.

Reply:
Why do you need to specify all of the statuses (which the board not supports yet, as far as i know, bull doesn't support waiting-children as well)

This interface must be a unify version between bullmq & bull

Reply:
I'm getting a build error unless I include them all.
https://gist.github.com/jamesgweber/cbf1e8bbf29f169964d37b6adb8c405a